static/app/views/settings/project/projectOwnership/addCodeOwnerModal.tsx

```diff
@@ -103,19 +113,41 @@ class AddCodeOwnerModal extends React.Component<Props, State> {
         <SourceFileBody>
           <IconCheckmark size="md" isCircled color="green200" />
           {codeownerFile.filepath}
-          <Button size="small" href={codeownerFile.html_url}>
+          <Button size="small" href={codeownerFile.html_url} target="_blank">
             {t('Preview File')}
           </Button>
         </SourceFileBody>
       </Panel>
     );
   }
 
-  errorMessage() {
-    const {errorJSON} = this.state;
+  errorMessage(baseUrl) {
+    const {errorJSON, codeMappingId} = this.state;
+    const {codeMappings} = this.props;
+    const codeMapping = codeMappings.find(mapping => mapping.id === codeMappingId);
+    const {integrationId, provider} = codeMapping as RepositoryProjectPathConfig;
     return (
       <Alert type="error" icon={<IconNot size="md" />}>
         <p>{errorJSON?.raw?.[0]}</p>
+        {codeMapping && (
+          <p>
+            {tct(
+              'Add the missing [userMappingsLink:User Mappings] and [teamMappingsLink:Team Mappings].',
```

Comment:
kind of a nit: it kind of feels weird that we tell them to add the missing user mappings if our error message just says the team names aren't associated. Maybe it could read something like 

"Configure User or Team mappings for any missing associations."